### PR TITLE
:memo: Enhance documentation

### PR DESCRIPTION
**Pull request by ghost**
Enhance documentation and add bower as a dependency in `package.json`.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,6 +4,7 @@
   "private": true,
   "devDependencies": {
     "brunch": "^2.0.1",
+    "bower": "^1.6.0",
```
Comment (notFloran) [inline package.json:7]: bower must be installed globally with `npm install -g bower`

Comment (nclsHart) [inline package.json:7]: Brunch must be installed globally too.

Comment (notFloran) [inline package.json:7]: Example : https://github.com/brunch/dead-simple

Comment (ghost) [inline package.json:7]: It annoys me that we don't have the real required dependencies in `package.json`. This is the goal of this file...

Mainly, I think that anyone should be able to install all the dependencies with

``` bash
$ npm install
```

Using `-g` is an _advanced_ way to do this.

Comment (notFloran) [inline package.json:7]: @nclsHart @martin-damien What do you think of this approach http://www.joezimjs.com/javascript/no-more-global-npm-packages/ ?

Comment (nclsHart) [inline package.json:7]: @notFloran It's a very interesting approach.

:+1:

Comment (ghost) [inline package.json:7]: :+1: too

**Pull request merged**